test/functional/apps.js

```diff
@@ -0,0 +1,35 @@
+import path from 'path';
+
+let testAppPath, uiCatalogApp;
+
+// Had to make these two optional dependencies so the tests
+// can still run in linux
+if (process.platform === 'darwin') {
+  testAppPath = require('ios-test-app').absolute;
+  uiCatalogApp = require('ios-uicatalog');
+}
+
+const apps = {};
+
+const {REAL_DEVICE, CLOUD} = process.env;
+
+if (REAL_DEVICE) {
+  if (CLOUD) {
+    apps.testAppId = 1;
+  } else {
+    apps.iosTestApp = testAppPath.iphoneos;
+    apps.uiCatalogApp = path.resolve('.', 'node_modules', 'ios-uicatalog', uiCatalogApp[0]);
+  }
+} else {
+  if (CLOUD) {
+    apps.iosTestApp = 'http://appium.github.io/appium/assets/TestApp7.1.app.zip';
```

Comment:
We should probably do a more recent build than Xcode 7.1 for these, no?

Reply:
Is there a way to decide which build to use when doing cloud testing? Or should we just use the most recent Xcode build?